src/controllers/signal-client.js

```diff
@@ -0,0 +1,12 @@
+import clients from '../models/client';
+
+export default function (client, message) {
+  const {receiverId, signalData} = message;
+  const senderId = client.id;
+  const receiver = clients.all.find(client => client.id === receiverId);
```

Comment:
If this is the only way its being used it makes more sense to have the client store as something hashable rather than searching the whole array each time.